src/meta/src/stream/stream_graph/schedule.rs

```diff
@@ -110,8 +110,6 @@ crepe::crepe! {
 
     // The downstream fragment of a `Simple` edge must be singleton.
     SingletonReq(y) <- Edge(_, y, Simple);
-    // The downstream fragment of a `CdcTablename` edge must be singleton.
-    SingletonReq(y) <- Edge(_, y, CdcTablename);
```

Comment:
So the cdc tables' `Materialize` can be distributed, right? But the price is, there is always an additional shuffle between `backfill -> mview`. Personally, I don't think it's worth to do.